x-pack/plugins/apm/public/components/shared/AgentIcon/get_agent_icon.ts

```diff
@@ -60,10 +61,15 @@ export function getAgentIconKey(agentName: string) {
   const lowercasedAgentName = agentName.toLowerCase();
 
   // RUM agent names
-  if (RUM_AGENT_NAMES.includes(lowercasedAgentName as AgentName)) {
+  if (isRumAgentName(lowercasedAgentName)) {
     return 'rum';
   }
 
+  // Java  agent names
+  if (isJavaAgentName(lowercasedAgentName)) {
+    return 'java';
+  }
+
```

Comment:
It looks like you could call `normalizeAgentName` here. And here it says `rum` and on `normalizeAgentName` it says `rum-js`, should we choose one?

Reply:
True, and I initially wanted to use `normalizeAgentName` here but instead opted for as few changes as possible.
Same wrt to the names. The current map (`agentIcons`) has them as `java` and `rum`, and I preferred not changing that.